Explorer/Assets/Scripts/ECS/Unity/Transforms/Components/CameraTransformSync.cs

```diff
@@ -0,0 +1,4 @@
+namespace DCL.SDKComponents.TransformSync.Components
+{
+    public struct CameraTransformSync { }
```

Comment:
I think it's totally fine to identify by a component but considering you always have one entity and this entity can be passed to the constructor, you could just process the single entity in the system instead of the query.